eng/pipelines/release-ext-azure-coding-agent.yml

```
@@ -0,0 +1,33 @@
+# Continuous deployment trigger
+trigger:
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - go.mod
+      - cli/azd/extensions/azure.coding-agent
+      - eng/pipelines/release-azd-extension.yml
+      - /eng/pipelines/templates/jobs/build-azd-extension.yml
+      - /eng/pipelines/templates/jobs/cross-build-azd-extension.yml
+      - /eng/pipelines/templates/variables/image.yml
+
+pr:
+  paths:
+    include:
+      - go.mod
+      - cli/azd/extensions/azure.coding-agent
+      - eng/pipelines/release-azd-extension.yml
+      - eng/pipelines/templates/steps/publish-cli.yml
+    exclude:
+      - cli/azd/docs/**
+
+extends:
+  template: /eng/pipelines/templates/stages/1es-redirect.yml
+  parameters:
+    stages:
+      - template: /eng/pipelines/templates/stages/release-azd-extension.yml
+        parameters: 
+          AzdExtensionId: azure.coding-agent
+          SanitizedExtensionId: azure.coding-agent
```

```suggestion
          SanitizedExtensionId: azure-coding-agent
```

This becomes the filename so I get rid of dots.